controllers/newsletter.js

```diff
@@ -0,0 +1,46 @@
+const newsletters = require("../models/newsletter");
+
+const addEmail = async (req, res) => {
+  try {
+    const a = await newsletters.subscribe(req.body);
+    return res.json({
+      message: "gotcha",
+      response: a,
+    });
+  } catch (err) {
+    logger.error("control failed");
+    return res.boom.badImplementation("not working");
```

Comment:
Can we be more descriptive about the error here?

Reply:
done